src/index.js

```diff
@@ -1,14 +1,40 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, currently included via a script, is required for this line to work
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
+const todoItems = [
+
+  {
+    description: 'Walk the Dog',
+    completed: 'false',
+    index: 0,
+  },
+  {
+    description: 'Iron the shirt',
+    completed: 'false',
+    index: 1,
+  },
+  {
+    description: 'Attend meeting',
+    completed: 'false',
+    index: 2,
+  },
+];
+
+const todoList = document.querySelector('.todo-list');
+
+const renderTodoList = (items) => {
+  for (let i = 0; i < todoItems.length; i += 1) {
+    const todoItem = document.createElement('div');
+    todoItem.classList.add('flex-item');
+    todoItem.innerHTML = `<div class="flex-container">
+      <div class="input">
+      <input type="checkbox" name="${items[i].description}" id="${items[i].description}">
+      <label for="${items[i].description}">${items[i].description}</label>
+      </div>
+      <i class="fas fa-ellipsis-v"></i>
+      </div>`;
+    todoList.appendChild(todoItem);
+    console.log(items[i]);
   }
```

Comment:
- [ ] [OPTIONAL] Prefer to use the new ES6 syntax of using a `forEach` or `map`. this will make your code more concise and readable.
something like this would be better
```suggestion
  todoItems.forEach((item) => {
    const todoItem = document.createElement('div');
    todoItem.classList.add('flex-item');
    todoItem.innerHTML = `<div class="flex-container">
      <div class="input">
      <input type="checkbox" name="${item.description}" id="${item.description}">
      <label for="${item.description}">${item.description}</label>
      </div>
      <i class="fas fa-ellipsis-v"></i>
      </div>`;
    todoList.appendChild(todoItem);
    console.log(todoItem);
  });
```